pkg/run/storage.go

```diff
@@ -64,6 +68,43 @@ func (r *RunStorageService) ensureBucketExists(ctx context.Context, bucket strin
 	return err
 }
 
+func (r *RunStorageService) triggerBucketNotifications(ctx context.Context, bucket string, key string, eventType v1.BucketNotificationType) error {
```

Comment:
```suggestion
func (r *RunStorageService) triggerBucketNotifications(ctx context.Context, bucket string, key string, eventType v1.BucketNotificationType)  {
```

Remove error return type here, as we don't return errors.